test/rulesets/Base/array.jl

```diff
@@ -1,3 +1,34 @@
+@testset "constructors" begin
+
+    # We can't use test_rrule here (as it's currently implemented) because the elements of
```

Comment:
Perhaps open an issue and link it here?
I'm not sure if it's worth addressing somehow but maybe worth discussing it